orc8r/cloud/go/swagger/combine_test.go

```diff
@@ -0,0 +1,121 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger_test
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"magma/orc8r/cloud/go/swagger"
+	"magma/orc8r/cloud/go/tools/combine_swagger/spec"
+	"magma/orc8r/cloud/go/tools/swaggergen/generate"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v2"
+)
+
+func init() {
+	_ = flag.Set("alsologtostderr", "true") // uncomment to view logs during test
+}
+
+var (
+	testdataDir    = "testdata"
+	cfgsDir        = filepath.Join(testdataDir, "configs")
+	commonFilepath = filepath.Join(testdataDir, "common/common.yml")
+	outFilepath    = filepath.Join(testdataDir, "out.yml")
+	goldenFilepath = filepath.Join(testdataDir, "out.yml.golden")
+
+	nErrsFromCombine = 5
+)
+
+// TestCombine tests the generated output against a golden file.
+// Golden file was generated by the combine_swagger command, then manually
+// compared to go-swagger's mixin output of the same files, ensuring
+// the golden file's correctness.
+//
+// go-swagger mixin: https://goswagger.io/usage/mixin.html
+func TestCombine(t *testing.T){
```

Comment:
As stated elsewhere -- we don't need to test `Combine` here, just `CombineStr`

Be sure to move/preserve the doc comment